gui/slick/interfaces/default/testRename.tmpl

```diff
@@ -43,13 +63,13 @@
     <thead>
         <tr class="seasonheader" id="season-$cur_ep_obj.season">
             <td colspan="4">
-                <br/>
+            	<br />
```

Comment:
no. your commit is adding the space.
-                <br/>
+            	<br />